src/compas_fab/backends/ros/messages/services.py

```diff
@@ -164,4 +166,52 @@ def __init__(self, planner_config='', group='', params=PlannerParams(), replace=
         self.planner_config = planner_config
         self.group = group
         self.params = params
-        self.replace = replace
\ No newline at end of file
+        self.replace = replace
+
+class MotionPlanRequest(ROSmsg):
+    """http://docs.ros.org/kinetic/api/moveit_msgs/html/msg/MotionPlanRequest.html
+    """
+    def __init__(self, workspace_parameters=WorkspaceParameters(), start_state=RobotState(), 
+                 goal_constraints=[], path_constraints=Constraints(), 
+                 trajectory_constraints=TrajectoryConstraints(), planner_id="",
+                 group_name="", num_planning_attempts=8, allowed_planning_time=2.,
+                 max_velocity_scaling_factor=1., max_acceleration_scaling_factor=1.):
+        self.workspace_parameters = workspace_parameters # moveit_msgs/WorkspaceParameters
+        self.start_state = start_state # moveit_msgs/RobotState 
+        self.goal_constraints = goal_constraints # moveit_msgs/Constraints[] 
+        self.path_constraints = path_constraints # moveit_msgs/Constraints 
+        self.trajectory_constraints = trajectory_constraints # moveit_msgs/TrajectoryConstraints 
+        self.planner_id = planner_id # string 
+        self.group_name = group_name # string 
+        self.num_planning_attempts = num_planning_attempts # int32 
+        self.allowed_planning_time = allowed_planning_time # float64 
+        self.max_velocity_scaling_factor = max_velocity_scaling_factor # float64 
+        self.max_acceleration_scaling_factor = max_acceleration_scaling_factor # float64
+    
+    @property
+    def msg(self):
+        msg = super(MotionPlanRequest, self).msg
+        return {"motion_plan_request":msg}
+
+class MotionPlanResponse(ROSmsg):
+    """http://docs.ros.org/kinetic/api/moveit_msgs/html/msg/MotionPlanResponse.html
+    """
+
+    def __init__(self, trajectory_start=RobotState(), group_name="", 
```

Comment:
Isn't it problematic to initialize like this? I think they all share the same instance across the entire program lifetime. It's safer to default to `None` and initialize inside the constructor to the specific default, e.g. `RobotState()`.
Unless these defaults are immutable, then it's not a problem to keep as is.